internal/quick/init.go

```diff
@@ -42,13 +46,29 @@ var InitCommand = &command.Command{
 		args []string,
 		globalFlags command.GlobalFlags,
 		services *services.Services,
+		proj *project.Project,
 	) (command.Result, error) {
-		proj, err := services.Project.Init(
+		sigAlgo := crypto.StringToSignatureAlgorithm(initFlag.ServiceKeySigAlgo)
```

Comment:
This also looks like the same code as here https://github.com/onflow/flow-cli/pull/286/files#diff-e012f7fa2218d10a3c80f965cd049a7cfbfb2e7c89a3c5fdf4c8b2fbdc24b0ecR73